### DISCUSSION TITLE
Wakatime error

### DISCUSSION THREAD

**Discussion by mbutsk** [anuraghazra/github-readme-stats#3828] [Q&A] [answered]
I am trying to make a WakaTime Stats Card. Error below 
[image](https://github.com/anuraghazra/github-readme-stats/assets/81034449/5ef78dee-2e8c-432a-8451-2c4e65b9d718)
Screenshot from settings
![image](https://github.com/anuraghazra/github-readme-stats/assets/81034449/e9aa4dad-6a68-4a6d-97b4-cde464a3a5e7)
I edited the code today. Even in the private Dashboard it persists

**Comment by qwerty541** [comment, marked as answer]
Hey @mbutsk. I just checked your profile and the card works well. Here is a screenshot:

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/55fa52a1-d840-4d0a-b995-21a48838cc3d)

If you just created a wakatime profile, it can take some time to accumulate and update the data. According to my own experience of using wakatime, they update the data available through their API once per day.
